### PR TITLE
fix(strings): updated misspelled strings and created missing ones

### PR DESCRIPTION
**Pull request by Odonata971**

### CODE REVIEW

File: local.build.gradle [hunk 1]
```diff
@@ -0,0 +1,8 @@
+android {
```
Comment (Francois389) [inline local.build.gradle]: This file should be ignored.

**Review by Francois389 — CHANGES_REQUESTED** [review]

File: local.build.gradle (hunk 1 above)
Comment (Odonata971) [reply local.build.gradle]: my bad, i put the local.build.gradle in the wrong directory, i deleted it

**Review by Francois389 — DISMISSED** [review]
LGTM

**Review by Francois389 — APPROVED** [review]
👍

**Pull request merged**